examples/recover/recover.go

```diff
@@ -0,0 +1,49 @@
+// A `recover` means recovering from a `panic`, either from a "business" or "built-in" panic.
+// We want to recover if we want to handle a panic, stopping it from propagating upwards.
+
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+
+	recoverFromBuiltInPanic(10)
+
+	fmt.Println()
+
+	recoverFromCustomPanic(-1)
+}
+
+func recoverFromBuiltInPanic(i int) {
```

Comment:
Not sure if having two almost identical functions for builtin and custom panics is really useful here.

All things being equal, the sample should be as small as possible. The goal here is to remember (or copy-paste) the proper invocation of `recover` inside a `defer`, not explore different ways in which Go can panic.

A simple "custom" example should suffice. The panic example has a panic, the recover example should just recover from the same panic.

Reply:
Fixed as you requested in 47eed5ec17c1925520196d718befa82e69f5ccbc.
Thanks for the feedback.

Reply:
But I didn't use the same panic, as I think it's important to see that the panic might happen in a "lower" function call.
If it's critical I can change to align with the panic example. Or perhaps align the panic example?
Whatever you prefer...